### PR TITLE
Update custom-properties.md

### PR DESCRIPTION
**Pull request by Kurok7**
Added a video under Assignment that could help clarify what was learned already in the lesson.

Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [ ] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [x] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [x] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

Closes #XXXXX

**2. This PR:**

**3. Additional Information:**

### CODE REVIEW

File: html_css/intermediate_css/custom-properties.md [hunk 1]
```diff
@@ -136,6 +136,7 @@ Using the `prefers-color-scheme` media query can be pretty helpful for users sin
 1. Read through MDN's [Using CSS custom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) page starting at the "Inheritance of custom properties" section.
 2. Open the inspector on this page to inspect the styles and see how Odin uses some custom properties.
 </div>
+3. [This video](https://www.youtube.com/watch?v=PHO6TBq_auI) is a great introduction to CSS custom properties. Go ahead and watch it.
```
Comment (01zulfi) [inline html_css/intermediate_css/custom-properties.md:139]: I like the video, although I believe it doesn't cover anything new that isn't already covered in the lesson and the assignment itself. Hence, the video isn't necessary to complete the lesson. Nevertheless, we could add it to the Additional Resources section of the lesson. What do you think about that?

Comment (Kurok7) [reply html_css/intermediate_css/custom-properties.md:139]: I think that works out just fine. Thank you.

Comment (01zulfi) [reply html_css/intermediate_css/custom-properties.md:139]: Could you make that change please?

File: html_css/intermediate_css/custom-properties.md [hunk 2]
```diff
@@ -139,7 +139,7 @@ Using the `prefers-color-scheme` media query can be pretty helpful for users sin
 
 ### Additional Resources
 
-This section contains helpful links to other content. It isn’t required, so consider it supplemental for if you need to dive deeper into something.
+1. [This video](https://www.youtube.com/watch?v=PHO6TBq_auI) is a great introduction to CSS custom properties. Go ahead and watch it.
```
Comment (01zulfi) [inline html_css/intermediate_css/custom-properties.md:142]: One minor change regarding link text. We should make the link text more descriptive here:

```suggestion
1. This [video on CSS custom properties](https://www.youtube.com/watch?v=PHO6TBq_auI) is a great introduction. Go ahead and watch it.
```

**Review by 01zulfi — CHANGES_REQUESTED** [review]

**Comment by thatblindgeye** [comment]
@Kurok7 following up to see if you'd like to make the above change request

**Review by zachmmeyer — APPROVED** [review]

**Pull request merged**